pep-0001.txt

```diff
@@ -696,7 +697,7 @@ For each new PEP that comes in an editor does the following:
 
 * The file name extension is correct (i.e. ``.rst``).
 
-* Make sure the appropriate core developer(s) is added to
+* Make sure the appropriate sponsor or co-author(s) are added to
```

Comment:
```suggestion
* Make sure the appropriate sponsor/co-author(s) are added to
```

Implement @Rosuav 's suggestion

Reply:
```diff
- * Make sure the appropriate core developer(s) is added to
-   `.github/CODEOWNERS` [7]_.
+ * Ensure that everyone listed as a sponsor or co-author of the PEP with write 
+   access to the `PEP repository`_ is added to `.github/CODEOWNERS` [7]_.
```

A larger rephrasing, but avoids a slash in prose. @Rosuav / @CAM-Gerlach ? I understand the plural agreement issue now (we could also solve it by saying sponsor(s), but I'm not sure it makes sense for there to be more than one sponsor on a PEP)

A

Reply:
Seems decent. It runs on a bit, though; what about "who has write access" to clarify that clause?